portfolio/src/main/webapp/script.js

```diff
@@ -86,3 +86,8 @@ function createListElement(text) {
     liElement.innerText = text;
     return liElement;
 }
+
+function deleteComments() {
```

Comment:
Please add documentation explaining that this will refetch comments list after deleting